crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -83,9 +83,11 @@ impl Extension for LoggerExtension {
             .filter(|(_, operation)| operation.node.ty == OperationType::Query)
             .any(|(_, operation)| operation.node.selection_set.node.items.iter().any(|selection| matches!(&selection.node, Selection::Field(field) if field.node.name.node == "__schema")));
         if !is_schema && self.config.log_request_query {
+            let query_id: &Uuid = ctx.data_unchecked();
+            let session_id: &SocketAddr = ctx.data_unchecked();
             info!(
-                query_id = ctx.data_unchecked::<Uuid>().to_string(),
-                session_id = ctx.data_unchecked::<SocketAddr>().to_string(),
+                query_id = %query_id,
+                session_id = %session_id,
```

Comment:
You should even be able to do the following:

```suggestion
                %query_id,
                %session_id,
```

Reply:
Ah, didn't think to try that :)